bin/scrub-recall-fixture.py

```
@@ -0,0 +1,151 @@
+NOUN = ["marmot","sprocket","lantern","gantry","harrow","kestrel","dulcimer","trellis","pergola",
+    "quokka","zephyr","obelisk","pangolin","axolotl","narwhal","dovetail","flywheel","capstan",
+    "gudgeon","tamarind","quince","mandrel","ferrule","gnomon"]
+VERB = {"directive":"be reconciled nightly","decision":"was chosen over the alternative",
+    "bug":"was mis-tallied on rollover","architecture":"routes through the staging shim",
+    "correction":"was restated after review","content":"is summarized for the digest"}
+
+def marker(i):  # unique bigram for index i (< 24*24 = 576)
+    return f"{ADJ[i % len(ADJ)]} {NOUN[(i // len(ADJ)) % len(NOUN)]}"
+
+def synth(fid, kind, marker_phrase, *, reps=1, filler=True):
+    dom = DOMAINS[int(hashlib.sha1(fid.encode()).hexdigest(), 16) % len(DOMAINS)]
```

Ruff (0.16.2)

[error] Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)